engine/robot_logic/traversal.py

```diff
@@ -347,3 +347,36 @@ def turn_to_target_heading(
     robot_state.enable_obstacle_avoidance = True
     if not robot_state.is_sim:
         robot_state.motor_controller.spin_motors(0, 0)
+
+    def ackerman_calculations (left_ang, right_ang, inside_ang, wheel_base, 
+                               steering_ratio):
+        #calculate ideal wheel angle
+
+        ackerman_percentage = 0 #This value needs to be determined by mechanical
```

Comment:
can we document what this is?

Reply:
Can we also make that a global variable since I assume it will stay constant